.travis.yml

```diff
@@ -82,7 +82,7 @@ before_script:
     nvm install 8
   - npm install
 script:
-  - phpunit -c phpunit.xml.dist
+  - phpunit -c phpunit.xml
```

Comment:
Why ? This now prevents overloading the phpunit config.

Reply:
Fixed in eb3165fea32e2f3bafde47e4bcf62cf92e008a15